Makefile

```diff
@@ -1,13 +1,43 @@
 OADP_TEST_NAMESPACE ?= openshift-adp
-REGION ?= us-east-1
-PROVIDER ?= aws
-CLUSTER_PROFILE ?= aws
+CLUSTER_TYPE ?= aws
+
+# CONFIGS FOR CLOUD
+OADP_CRED_DIR ?= /var/run/oadp-credentials # bsl / blob storage cred dir
+CLUSTER_PROFILE_DIR ?= /Users/drajds/.aws # vsl / volume/cluster cred dir
+
+OADP_CRED_FILE ?= ${OADP_CRED_DIR}/new-aws-credentials # bsl cred file
+CI_CRED_FILE ?= ${CLUSTER_PROFILE_DIR}/.awscred # vsl cred file
+
+# aws configs - default
+BSL_REGION ?= us-east-1
+VSL_REGION ?= ${LEASED_RESOURCE}
+BSL_AWS_PROFILE ?= migration-engineering
+
+# vsl secret
 CREDS_SECRET_REF ?= cloud-credentials
-OADP_AWS_CRED_FILE ?= /var/run/oadp-credentials/aws-credentials
-OADP_S3_BUCKET ?= /var/run/oadp-credentials/velero-bucket-name
+# bucket file
+OADP_BUCKET_FILE ?= ${OADP_CRED_DIR}/new-velero-bucket-name
+# azure cluster resource file - only in CI
+AZURE_RESOURCE_FILE ?= /var/run/secrets/ci.openshift.io/multi-stage/metadata.json
+
+# Misc
+OPENSHIFT_CI ?= true
 VELERO_INSTANCE_NAME ?= velero-sample
 E2E_TIMEOUT_MULTIPLIER ?= 1
 
+ifeq ($(CLUSTER_TYPE), gcp)
+	CI_CRED_FILE = ${CLUSTER_PROFILE_DIR}/gce.json
+	OADP_CRED_FILE = ${OADP_CRED_DIR}/gcp-credentials
+	CREDS_SECRET_REF = cloud-credentials-gcp
+	OADP_BUCKET_FILE = ${OADP_CRED_DIR}/gcp-velero-bucket-name
+else ifeq ($(CLUSTER_TYPE), azure4)
```

Comment:
azure4, is that the real cluster type or typo?

Reply:
Yes, thats the real cluster type @weshayutin